examples/all-clusters-app/all-clusters-common/src/camera-av-stream-delegate-impl.cpp

```diff
@@ -217,7 +217,7 @@ void CameraAVStreamManager::OnAttributeChanged(AttributeId attributeId)
     ChipLogProgress(Zcl, "Attribute changed for AttributeId = " ChipLogFormatMEI, ChipLogValueMEI(attributeId));
 }
 
-Protocols::InteractionModel::Status CameraAVStreamManager::CaptureSnapshot(const uint16_t streamID,
+Protocols::InteractionModel::Status CameraAVStreamManager::CaptureSnapshot(const chip::app::DataModel::Nullable<uint16_t> streamID,
```

Comment:
Please get rid of the extra chip:: etc prefixes, in all this stuff that has `using` bits.